homeassistant/components/honeywell/__init__.py

```diff
@@ -44,12 +44,12 @@ def _async_migrate_data_to_options(
     )
 
 
-async def async_setup_entry(hass: HomeAssistant, config: ConfigEntry) -> bool:
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
```

Comment:
Maybe I'm getting carried away, but if you rename `config` > `config_entry` for ConfigEntry parameters, then I think you should do it on `update_listener` and `async_unload_entry` also.

And while you are at it, could you add type hints to `get_somecomfort_client` and `HoneywellData.__init__`

Reply:
Haha for sure, I can do a clean up here.

Reply:
After taking a look at the `somecomfort` library to confirm the actual types that we would see in HASS. Many of them are `Any` so I used that in HASS as well. One day I will put some effort into modernizing that `somecomfort` library.

Reply:
Even though the low-level library doesn't use type hints, it's completely reasonable to add `str` type hints for those here. Adding `Any` is really no different than not having hints at all.

Reply:
Done.